service/src/main/java/bio/terra/policy/service/pao/PaoService.java

```diff
@@ -91,6 +92,11 @@ public Pao getPao(UUID objectId) {
     return paoDao.getPao(objectId);
   }
 
+  @ReadTransaction
```

Comment:
This works and I see it's done below for `linkSourcePao()` but IMO I would instead add a public `listPaos()` to `PaoDao` and add the transaction there, rather than putting it in the service code.

Reply:
I agree the current transaction boundary is a problem, but I intend to go the other way and move all transactions to the service layer.